infrastructure/ansible/roles/bot/defaults/main.yml

```diff
@@ -19,23 +19,11 @@ bot_max_memory: "196m"
 # Each bot number matches a bot instance that will be deployed and started.
 bot_numbers: ["01", "02", "03", "04" ]
 
-# Bot prefix we expect to be overriden in inventory.
-# We deploy bots to multiple server, to keep the host numbers unique, there is a prefix.
-# The bot prefix should be unique across all bot hosts.
-# EG: with bot_prefix = 1, and bot_number=01, the bot number in the bot name will be '101'.
-bot_prefix: ""
-
-# Name of the bot server, this will typically be overriden in inventory. The final name of the bot host is
-# concatenation of the "Bot" the "bot name", and the "bot number" with prefix.
-bot_name: "Server"
-
-# Arguments when running bot servers. Each of the key value pairs will be added as a '-p{key}={value}' argument
-# passed to the JVM when running the bot.
-bot_run_args:
```

Comment:
bot_run_args was used in 'run_server', we inline these variables directly  to the 'bot.service' file